packages/simcore-sdk/src/simcore_sdk/node_ports_common/r_clone.py

```diff
@@ -90,7 +90,9 @@ async def _async_r_clone_command(
     )
 
     assert proc.stdout  # nosec
-    await asyncio.wait([_read_stream(proc.stdout, [*r_clone_log_parsers])])
+    await asyncio.wait(
```

Comment:
See: https://docs.python.org/3/library/asyncio-task.html#asyncio.wait

> Changed in version 3.11: Passing coroutine objects to wait() directly is forbidden.

Reply:
I had this question above, now I see the answer here. nice. thx!